src/neo/Cryptography/VRF.cs

```diff
@@ -0,0 +1,373 @@
+using System.Numerics;
+using System.Security.Cryptography;
+using System;
+using System.Linq;
+namespace Neo.Cryptography
+{
+    /// <summary>
+    /// Verifiable Random Function
+    /// Based on VRF draft-irtf-cfrg-vrf-08
+    /// 
+    /// VRF function consumes an octet string as input
+    /// and generates a verifiable random number during the consensus.
+    /// </summary>
+    public static class VRF
```

Comment:
Is this class only used in the plugin? If so, I think it should be moved to the dBFT plugin.

Reply:
Moved to dbft plugin